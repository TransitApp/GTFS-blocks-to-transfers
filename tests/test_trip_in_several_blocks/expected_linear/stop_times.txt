trip_id,stop_id,stop_sequence,arrival_time,departure_time
trip_3,junction,0,24:00:00,24:00:00
trip_3,slocan-park,1,24:25:00,24:25:00
trip_3,slocan-city,2,24:55:00,24:55:00
trip_4,castlegar-tc,0,20:00:00,20:00:00
trip_4,junction,1,20:25:00,20:25:00
trip_4,nelson-tc,2,20:50:00,20:50:00
trip_5,nelson-tc,0,21:00:00,21:00:00
trip_5,bonnington,1,21:25:00,21:25:00
trip_5,junction,2,21:50:00,21:50:00
trip_1_b2t:if_mon-tues-wed-thurs,junction,0,22:00:00,22:00:00
trip_1_b2t:if_mon-tues-wed-thurs,slocan-park,1,22:25:00,22:25:00
trip_1_b2t:if_mon-tues-wed-thurs,slocan-city,2,22:55:00,22:55:00
trip_1_b2t:if_fri-sat,junction,0,22:00:00,22:00:00
trip_1_b2t:if_fri-sat,slocan-park,1,22:25:00,22:25:00
trip_1_b2t:if_fri-sat,slocan-city,2,22:55:00,22:55:00
trip_2_b2t:if_fri-sat,slocan-city,0,23:00:00,23:00:00
trip_2_b2t:if_fri-sat,slocan-park,1,23:25:00,23:25:00
trip_2_b2t:if_fri-sat,junction,2,23:55:00,23:55:00
trip_1_b2t:if_sun,junction,0,22:00:00,22:00:00
trip_1_b2t:if_sun,slocan-park,1,22:25:00,22:25:00
trip_1_b2t:if_sun,slocan-city,2,22:55:00,22:55:00
trip_2_b2t:if_sun,slocan-city,0,23:00:00,23:00:00
trip_2_b2t:if_sun,slocan-park,1,23:25:00,23:25:00
trip_2_b2t:if_sun,junction,2,23:55:00,23:55:00
