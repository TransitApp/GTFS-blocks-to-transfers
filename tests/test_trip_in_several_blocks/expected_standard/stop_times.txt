trip_id,stop_sequence,stop_id,arrival_time,departure_time
trip_1,0,junction,22:00:00,22:00:00
trip_1,1,slocan-park,22:25:00,22:25:00
trip_1,2,slocan-city,22:55:00,22:55:00
trip_2,0,slocan-city,23:00:00,23:00:00
trip_2,1,slocan-park,23:25:00,23:25:00
trip_2,2,junction,23:55:00,23:55:00
trip_3,0,junction,24:00:00,24:00:00
trip_3,1,slocan-park,24:25:00,24:25:00
trip_3,2,slocan-city,24:55:00,24:55:00
trip_4,0,castlegar-tc,20:00:00,20:00:00
trip_4,1,junction,20:25:00,20:25:00
trip_4,2,nelson-tc,20:50:00,20:50:00
trip_5,0,nelson-tc,21:00:00,21:00:00
trip_5,1,bonnington,21:25:00,21:25:00
trip_5,2,junction,21:50:00,21:50:00
