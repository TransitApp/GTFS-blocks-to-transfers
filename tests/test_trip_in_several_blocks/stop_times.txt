trip_id,stop_sequence,stop_id,arrival_time,departure_time
trip_1,0,slocan_city,22:00:00,22:00:00
trip_1,1,slocan_park,22:25:00,22:25:00
trip_1,2,junction,22:55:00,22:55:00
trip_2,0,junction,23:00:00,23:00:00
trip_2,1,slocan_park,23:25:00,23:25:00
trip_2,2,slocan_city,23:55:00,23:55:00
trip_3,0,slocan_city,24:00:00,24:00:00
trip_3,1,slocan_park,24:25:00,24:25:00
trip_3,2,junction,24:55:00,24:55:00
trip_4,0,slocan_city,20:00:00,20:00:00
trip_4,1,slocan_park,20:25:00,20:25:00
trip_4,2,junction,20:50:00,20:50:00
trip_5,0,junction,21:00:00,21:00:00
trip_5,1,slocan_park,21:25:00,21:25:00
trip_5,2,slocan_city,21:50:00,21:50:00
vs_1,0,slocan_city,08:00:00,08:00:00
vs_1,1,slocan_park,08:25:00,08:25:00
vs_2,0,slocan_park,08:26:00,08:26:00
vs_2,1,storbo,08:27:00,08:27:00
vs_2,2,slocan_park,08:28:00,08:28:00
vs_3,0,slocan_park,08:30:00,08:30:00
vs_3,1,slocan_city,08:55:00,08:55:00
cycle_1,0,slocan_city,05:00:00,05:00:00
cycle_1,1,junction,05:30:00,05:30:00
cycle_2,0,junction,11:00:00,11:00:00
cycle_2,1,slocan_city,11:30:00,11:30:00
cycle_3,0,slocan_city,24:01:00,24:01:00
cycle_3,1,junction,24:31:00,24:31:00
cycle_4,0,junction,25:02:00,25:02:00
cycle_4,1,slocan_city,25:32:00,25:32:00
ws_1,0,slocan_city,07:55:00,07:55:00
ws_1,1,slocan_park,08:20:00,08:20:00
ws_2,0,slocan_park,08:26:00,08:26:00
ws_2,1,storbo,08:27:00,08:27:00
ws_2,2,slocan_park,08:28:00,08:28:00
multi_1,0,slocan_city,22:00:00,22:00:00
multi_1,1,slocan_park,22:25:00,22:25:00
multi_1,2,junction,22:55:00,22:55:00
multi_2F,0,junction,23:00:00,23:00:00
multi_2F,1,slocan_park,23:25:00,23:25:00
multi_2F,2,slocan_city,23:55:00,23:55:00
multi_2B,0,junction,23:00:00,23:00:00
multi_2B,1,slocan_park,23:25:00,23:25:00
multi_2B,2,slocan_city,23:55:00,23:55:00
multi_3,0,slocan_city,24:00:00,24:00:00
multi_3,1,slocan_park,24:25:00,24:25:00
multi_3,2,junction,24:55:00,24:55:00
multi_5,0,junction,21:00:00,21:00:00
multi_5,1,slocan_park,21:25:00,21:25:00
multi_5,2,slocan_city,21:50:00,21:50:00
multi_0,0,junction,21:00:00,21:00:00
multi_0,1,slocan_park,21:25:00,21:25:00
multi_0,2,slocan_city,21:50:00,21:50:00
