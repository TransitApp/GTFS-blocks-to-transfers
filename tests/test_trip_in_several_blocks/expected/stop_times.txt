trip_id,stop_id,stop_sequence,arrival_time,departure_time
cycle_1,slocan_city,0,05:00:00,05:00:00
cycle_1,junction,1,05:30:00,05:30:00
cycle_2,junction,0,11:00:00,11:00:00
cycle_2,slocan_city,1,11:30:00,11:30:00
cycle_3,slocan_city,0,24:01:00,24:01:00
cycle_3,junction,1,24:31:00,24:31:00
cycle_4,junction,0,25:02:00,25:02:00
cycle_4,slocan_city,1,25:32:00,25:32:00
multi_0,junction,0,21:00:00,21:00:00
multi_0,slocan_park,1,21:25:00,21:25:00
multi_0,slocan_city,2,21:50:00,21:50:00
multi_1,slocan_city,0,22:00:00,22:00:00
multi_1,slocan_park,1,22:25:00,22:25:00
multi_1,junction,2,22:55:00,22:55:00
multi_2B,junction,0,23:00:00,23:00:00
multi_2B,slocan_park,1,23:25:00,23:25:00
multi_2B,slocan_city,2,23:55:00,23:55:00
multi_2F,junction,0,23:00:00,23:00:00
multi_2F,slocan_park,1,23:25:00,23:25:00
multi_2F,slocan_city,2,23:55:00,23:55:00
multi_3,slocan_city,0,24:00:00,24:00:00
multi_3,slocan_park,1,24:25:00,24:25:00
multi_3,junction,2,24:55:00,24:55:00
multi_5,junction,0,21:00:00,21:00:00
multi_5,slocan_park,1,21:25:00,21:25:00
multi_5,slocan_city,2,21:50:00,21:50:00
trip_1,slocan_city,0,22:00:00,22:00:00
trip_1,slocan_park,1,22:25:00,22:25:00
trip_1,junction,2,22:55:00,22:55:00
trip_2,junction,0,23:00:00,23:00:00
trip_2,slocan_park,1,23:25:00,23:25:00
trip_2,slocan_city,2,23:55:00,23:55:00
trip_3,slocan_city,0,24:00:00,24:00:00
trip_3,slocan_park,1,24:25:00,24:25:00
trip_3,junction,2,24:55:00,24:55:00
trip_4,slocan_city,0,20:00:00,20:00:00
trip_4,slocan_park,1,20:25:00,20:25:00
trip_4,junction,2,20:50:00,20:50:00
trip_5,junction,0,21:00:00,21:00:00
trip_5,slocan_park,1,21:25:00,21:25:00
trip_5,slocan_city,2,21:50:00,21:50:00
vs_1,slocan_city,0,08:00:00,08:00:00
vs_1,slocan_park,1,08:25:00,08:25:00
vs_2,slocan_park,0,08:26:00,08:26:00
vs_2,storbo,1,08:27:00,08:27:00
vs_2,slocan_park,2,08:28:00,08:28:00
ws_1,slocan_city,0,07:55:00,07:55:00
ws_1,slocan_park,1,08:20:00,08:20:00
ws_2,slocan_park,0,08:26:00,08:26:00
ws_2,storbo,1,08:27:00,08:27:00
ws_2,slocan_park,2,08:28:00,08:28:00
vs_3_b2t:if_b2t:service_0,slocan_park,0,08:30:00,08:30:00
vs_3_b2t:if_b2t:service_0,slocan_city,1,08:55:00,08:55:00
vs_3_b2t:if_sun,slocan_park,0,08:30:00,08:30:00
vs_3_b2t:if_sun,slocan_city,1,08:55:00,08:55:00
